### ISSUE TITLE
Banner & Nav style for staff authored landing pages

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#2542]
From Matthew: "Some campaigns and program initiatives warrant a landing page that's a step above our current staff authored pages. The page should establish a presence as the home for these long-term efforts. Example EU MisInfo campaign. My gut tells me that key to achieve this will be an image that appears outside/above/behind the content box. Horizontal nav might also help signal this. We could limit access to this template to official initiatives with engagement team approval to ensure quality and maintain a hierarchy between primary site pages and staff/fellow content."

Basically, I think we need something that is in between (in visually hierarchy and IA) a primary page like About and a tertiary multi-page. This _could_ be as simple as updating the black/white marble background with a campaign image.
![red-pen-87c10e69-2bbb-4fd8-c285-f3db1a466272](https://user-images.githubusercontent.com/1068783/51944664-78015100-23d1-11e9-9ba2-43369d1ec57b.jpg)

This work could be done in unison with https://github.com/mozilla/foundation.mozilla.org/issues/2275

cc @xmatthewx

**Comment by kristinashu** [comment]
Assigning to @beccaklam to gather more info, like how many pages a campaign would have, how much content, and what type of content.

**Comment by beccaklam** [comment]
More info on Misinfo campaign deliverables: https://docs.google.com/document/d/1BMMghJIUMuMBZojQB-rILLDOu7oxZ_A8XGk2Xar-wQc/edit?usp=sharing

**Comment by beccaklam** [comment]
## Requested Deliverables (in order of public launch)

### Feb 11

- [ ] Open Letter (https://github.com/MozillaFoundation/Design/issues/324)

- [ ] Mozilla Campaign Page

- [ ] Unique Vanity URL (for the Campaign Page?)

### March 1

- [ ] Scorecard

- [ ] Resources for partners

### March 4

- [ ] Tools (election bundle) _(MoCo is creating this)_

- [ ] Mitchel op-ed

**Comment by kristinashu** [comment]
I know you're trying, but we need to know what type of content the campaign page will have and what sections. Does the petition always need to be present or only on one of the sub pages? Are those deliverables each pages on the site? But I know you are trying to get content and haven't been able to.

**Comment by kristinashu** [comment]
Lol yeah. I suspect that for now we can just use the multi-page campaign template like https://foundation.mozilla.org/en/campaigns/aadhaar/ (would have to use screen caps for the score card table). 
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/1068783/52302403-e7cb8a80-2941-11e9-96f5-cbef8bed2704.png">

And then as we get content and a better idea of what's happen we can spruce it up. Exploring stuff like this but really can't proceed until we know more.

<img width="996" alt="image" src="https://user-images.githubusercontent.com/1068783/52302481-2e20e980-2942-11e9-9dca-8d91e103db62.png">

**Comment by beccaklam** [comment]
@kristinashu I don't think the scorecard needs to be anything more than a downloadable pdf on the campaign site. The site they've given us as an example just has a place where people can download resources: https://blueprintsfc.org/
It doesn't sound like they need anything drastically different that would require us to create a different template. Unless I'm missing something?

**Comment by kristinashu** [comment]
No! I'm missing everything lol. PDF sounds good!

**Comment by xmatthewx** [comment]
I like how you've treated a hero and CTA up top. It remains distinct from other top level pages, but feels like a legit landing. This feels like a great iteration on what we've got. 

For content below the nav, I think we can assume it's no more unique than what's on current campaign pages – we can tackle special snowflakes separately as they arise, but keeping that stuff at page level below the nav.

On a subpage, could top collapse to slim image and title? Feels like a lot of vertical.

**Comment by kristinashu** [comment]
Yeah the vertical is exactly where I started to get stuck. I like the idea of a collapsed version for other page tho. Will explore!

**Comment by xmatthewx** [comment]
I could imagine a half-height graphic with white cutout and page title to maintain visual connection with landing page.

**Comment by kristinashu** [comment]
@beccaklam @xmatthewx here are comps https://redpen.io/p/fzd8ac96a0d2afba37 (can't say I love any of them lol).

cc @natalieworth @sabrinang TLDR is that campaigns team would like a mini-site for their misinfo campaign. They feel the [existing multi-page](https://foundation.mozilla.org/en/campaigns/aadhaar/) campaign sites don't feel special or impactful enough and that the nav is too hidden.

**Comment by kristinashu** [comment]
I'm going to hold off on this until we have more info from campaign team on what their key CTA items are and how many pages we will have. Rebecca will gather info and sketch up a couple options. Then we can weigh ux vs dev time for each option.

**Comment by beccaklam** [comment]
**From the Misinfo Working Group on Thursday, Feb 14:**

The Campaigns team were originally thinking the nav of the campaign site would be:
- Scorecard
- Take Action
- Disinformation Resources

However, there is a desire for platform specific pages that explain how a given platform was evaluated along with a specific action to that platform for users to take. That may mean 2-3 of these pages. We need to think to see if there's an IA that could encompass this desire and that would look like. We also need a page that explains the methodology behind our scorecard standards. I'm thinking that could be a separate page or it could live under our scorecard.

**Comment by beccaklam** [comment]
Ok, 3 ideas for Misinfo Campaign IA (excuse my scribbly sketches)

![img_0421](https://user-images.githubusercontent.com/20977376/52885623-d6536100-313f-11e9-8676-4bc75df6395d.jpg)

**1.** We use an Opportunities template so that we can take advantage of the sub page capability the side nav has. Platform specific pages can live as sub pages on the 'Take Action' child page. May require some engineering so that we can get petition snippets on these pages.

**2.** We stick with the Campaigns template but make separate campaign pages for platform specific pages. We link out to these pages from the Take Action page and the Scorecard. 

**3.** We keep the Campaigns template but instead of a Take Action page we have platform specific pages in the nav. I wonder if this helps to simplify the campaign for people who can kind of see the nav as their own 'choose your own adventure'. 

CC: @kristinashu @xmatthewx @alanmoo

**Comment by alanmoo** [comment]
The Opportunities template nested nav isn't particularly good, so this might be ~an oppor~ a point at which we decide to improve functionality here, though I recognize we're limited by time.

Are you saying that having a "take action" page is negotiable (it could exist but we could have direct pages for the different platforms in the nav instead)?

IMO option 3 feels the grossest from an IA standpoint, and likely stretches the capabilities of what the campaign template can handle from a horizontal spacing standpoint.

This feels like the type of work that's worth the effort, and options 1&2 are similar technical lifts (adding a petition field to opportunity pages isn't clean, it'd probably end up being a new template anyway. But then we might hit layout issues with nav/content/petition all next to one another), so what do you like the best?

FWIW: I'm a fan of [these comps](https://redpen.io/p/fzd8ac96a0d2afba37) and they likely wouldn't require a ton of lift outside of figuring out our nested nav situation. The rest would mostly be stitching together elements of other templates into a new one.

**Comment by kristinashu** [comment]
@beccaklam do you have any content for this yet? Curious how much copy each section would have and if simple jump links or accordions could work instead of separate pages?

**Comment by beccaklam** [comment]
> @beccaklam do you have any content for this yet? Curious how much copy each section would have and if simple jump links (page anchors) or accordions could work instead of separate pages?

@kristinashu No content yet. The only problem with page anchors or accordions is if the campaigns team needs an action (like sign a petition) for each platform then having multiple snippets on one page would not be ideal?

**Comment by beccaklam** [comment]
> The Opportunities template nested nav isn't particularly good, so this might be an oppor a point at which we decide to improve functionality here, though I recognize we're limited by time.
> 
> Are you saying that having a "take action" page is negotiable (it could exist but we could have direct pages for the different platforms in the nav instead)?
> 
> IMO option 3 feels the grossest from an IA standpoint, and likely stretches the capabilities of what the campaign template can handle from a horizontal spacing standpoint.
> 
> This feels like the type of work that's worth the effort, and options 1&2 are similar technical lifts (adding a petition field to opportunity pages isn't clean, it'd probably end up being a new template anyway. But then we might hit layout issues with nav/content/petition all next to one another), so what do you like the best?
> 
> FWIW: I'm a fan of [these comps](https://redpen.io/p/fzd8ac96a0d2afba37) and they likely wouldn't require a ton of lift outside of figuring out our nested nav situation. The rest would mostly be stitching together elements of other templates into a new one.

I think we have the ability here to suggest to the campaigns team what the best IA would be. Therefore, a 'Take Action' page may not be completely necessary but I agree with you, option 3 is the least ideal option.

Both option 1 or 2 work for me. Option 1 makes it seem like the platform campaigns are part of the Misinfo campagin. Whereas option 2 makes it seem like the platform campaigns are entities of themselves part of a larger fight against Misinformation. I think option 2 we can already build without any engineering? So it depends if we want to put the work into option 1. Also, at this point the campaigns team doesn't have the actions ready yet so I wonder if it's even worth it to put in the work? (By that I mean, in the end if we end up with one action then it might not have been worth the effort ... unless we foresee a use case down the road, which we might)

**Comment by alanmoo** [comment]
As long as we're good with a single level nav, then yes, option 2 should be do-able without engineering.

**Comment by xmatthewx** [comment]
@beccaklam - thanks for putting pen to paper on this! @alanmoo @kristinashu - I'll work with @beccaklam to sort things out a bit further before you need to look too closely. 

@beccaklam - I think it's unlikely the team will run more than one action at a time (based on experience and the working group conversation). I expect that the main campaign page will contain the current main action. They could archive action content as a resource doc, but I don't think we need to keep up old petitions. 

The way I heard it, the bigger concern is containing lots of detailed info on each platform. 

Similar to your Option 2. The Scorecard will ideally summarize findings for 90% of visitors. From there, nerds can tap through to a page dedicated to each platform. These could be a page with left hand nav. We can put a fat button at the top that links back to primary landing page. It's not ideal UX, but will hopefully impact policy wonks, researchers, and others who can detail with jumping around. 

I'm thinking something like this URL-wise
 - campaign/eu-misinfo
 - campaign/eu-misinfo/resources
 - campaign/eu-misinfo/scorecard
 - campaign/eu-misinfo-facebook
 - campaign/eu-misinfo-facebook/something-specific
 - campaign/eu-misinfo-facebook/something-else-specific

It might be help to have Jon & Co start to draft pages as slides, so they start to think in terms of constrained pages.

**Comment by beccaklam** [comment]
@xmatthewx That makes sense? I've mocked up some comps to sketch out what we're thinking: https://redpen.io/p/az6289e0c6e14536b3 Let me know if what I'm mocking up matches what you think.

**Comment by beccaklam** [comment]
Invision flow of above sketches: https://mozilla.invisionapp.com/share/W7QMF4Y8AST#/348013962_Misinfo

**Comment by kristinashu** [comment]
Super useful to see the prototype and your latest comps.

See what @xmatthewx thinks but if the Google Action page is kinda like a sub-page of the main Misinfo pages then perhaps Misinfo should be a [primary page](https://foundation.mozilla.org/en/awards/) (but with unique header) and Action pages could be [normal campaign](https://redpen.io/jg336970d170fdeebb) or opportunity page (if no petition).

**Comment by beccaklam** [comment]
Staging link here for current rendition of the campaign site: https://foundation.mofostaging.net/en/campaigns/eu-misinformation/ Asking Brandi and Jon for feedback.

**Comment by alanmoo** [comment]
Remember that staging will get reset on Monday morning to match production.

**Comment by beccaklam** [comment]
Yeah I thought about that as I posted. So here is the updated [redpen](https://redpen.io/p/az6289e0c6e14536b3) and [invision prototype](https://mozilla.invisionapp.com/share/UZQN9GIXRB5#/348266393_Eumisinfo for better documentation.

**Comment by beccaklam** [comment]
Ok, so this hasn't been updated in a while. But what happened is the campaigns team didn't think the airtable scorecard/current campaigns template was impactful enough so switched course and made these designs: https://redpen.io/p/tafd9bccd2cd92a146

The plan is to create a Bannered Campaign template #2725 to launch the campaign with but eventually replace it with this design:
![image](https://user-images.githubusercontent.com/20977376/53741541-a8229080-3e64-11e9-8325-56490ee764da.png)

**Comment by xmatthewx** [comment]
The current solution ready for build is in the screenshot below. Design with side-by-side text and graphic needs more exploration with responsive and various use cases (including non-campaign things like mozfest or machine decision making) ... before we commit to that direction. We can discuss at next team huddle.  

<img width="1240" alt="image" src="https://user-images.githubusercontent.com/53411/53746657-ae6a3a00-3e6f-11e9-92e9-738a79352ed3.png">

**Comment by beccaklam** [comment]
@xmatthewx do you know who I should reach out to get some content for those use cases? (mozfest and machine decision making)

**Comment by xmatthewx** [comment]
TBD!! Let's discuss at our team huddle. We've got lots of stuff in the pipeline. I'd like to reset as a group and discuss how we want to approach all the things.

**Comment by kristinashu** [comment]
I'm gonna close this but please feel free to open back up if you think it's needed.

Misinfo has a nice mvp solution and we're aiming to find something that will work for special campaigns and also mozfest so let's see how the mozfest landing page shapes up and go from there.
